functional-tests/hoverctl/config_test.go

```diff
@@ -17,13 +17,13 @@ var _ = Describe("When I use hoverctl", func() {
 		})
 
 		It("prints the contents of the config.yaml", func() {
-			functional_tests.Run(hoverctlBinary, "targets", "create")
+			functional_tests.Run(hoverctlBinary, "targets", "new", "config-test")
```

Comment:
I changed it to new to be similar to the flag --new-target of which it is an alternative to. Pretty sure this was part of the changes we spoke about and agreed to last week, having the targets stuff looking similar to the flags that can be used on `hoverctl start` and `hoverctl login`.